seal_fhe/src/key_generator.rs

```diff
@@ -418,6 +425,20 @@ impl Clone for SecretKey {
     }
 }
 
+impl core::fmt::Debug for SecretKey {
```

Comment:
I guess this isn't the worst thing, but I have seen security issues at previous companies where people accidentally log credentials or secrets. Is it better to impl Debug and print **** as you've done or to not impl it at all?

Reply:
My personal opinion is that it is better to do this. For one, you all but eliminate the chances that someone will accidentally add a `#[derive(Debug)]`, which is the most likely way that someone will accidentally log the secret. Secondly, for convenience, since things like tests, assertions, etc. rely on a `Debug` impl.

But in this case it really doesn't matter, because the derived impl would just print a pointer value :upside_down_face:. So I did this for convenience, as there is no negative as far as I can see.